tsconfig.json

```diff
@@ -1,17 +1,47 @@
 {
   "compileOnSave": true,
   "compilerOptions": {
-    "forceConsistentCasingInFileNames": true,
-    "lib": ["dom", "es2017"],
-    "module": "commonjs",
+    // Enables all strict type checking options.
     "strict": true,
+
+    // Library files to be included in the compilation. Basically which "core
+    // language features" TypeScript should enable.
+    "lib": ["es2015", "es2016", "es2017"],
+
+    // Which version of EcmaScript TypeScript should transpile to. Because of
+    // how Babel and Webpack is set up we currently target `esnext`, and let
+    // Babel take care of transpiling it down.
+    "target": "esnext",
+
+    // Ideally we should output `esnext` modules, but because of how default
+    // exports have historically been handled in TypeScript we can't do it yet.
+    // However, https://github.com/Microsoft/TypeScript/pull/19675 is targeted
+    // for the next release of TypeScript, which means we should be able to
+    // switch this to `esnext` and specify `--esModuleInterop` to have it work
+    // as expected.
+    "module": "commonjs",
+
+    // Resolve modules in the same as Node.js. Aka make `require` works the same
```

Comment:
__typo:__ `in the same as Node.js` ---> `in the same way as Node.js` or something like this maybe?